### PR TITLE
ACMS-1510: Fixing the redirection issue with TFA module.

### PR DESCRIPTION
**Pull request by apathak18**
**Motivation**

Fixes ACMS-1510

**Proposed changes**

**Alternatives considered**

**Testing steps**

**Merge requirements**
- [ ] _Major change_, _Minor change_, _Bug_, _Enhancement_, and/or _Chore_ label applied
- [ ] Manual testing by a reviewer

### CODE REVIEW

**Review by rajeshreeputra — APPROVED** [review]
Looks good to me.

File: modules/acquia_cms_common/src/RedirectHandler.php [hunk 1]
```diff
@@ -89,7 +89,7 @@ public static function create(ContainerInterface $container) {
    * @param \Drupal\Core\Form\FormStateInterface $form_state
    *   The current form state.
    */
-  public static function submitForm(array &$form, FormStateInterface $form_state) : void {
+  public static function submitForm(array &$form, FormStateInterface $form_state): void {
     \Drupal::classResolver(static::class)->handleRedirect($form_state);
   }
```
Comment (chandan-singh7929) [inline modules/acquia_cms_common/src/RedirectHandler.php:94]: @apathak18 - I don't think we need this separate function, can we move all code from function `handleRedirect` to `submitForm`

Comment (apathak18) [reply modules/acquia_cms_common/src/RedirectHandler.php:94]: @chandan-singh7929 submitForm is static method called from acquia_cms_common_form_user_login_form_alter without discussion we cannot remove handleRedirect method.

**Review by chandan-singh7929 — APPROVED** [review]
LGTM

**Pull request merged**